src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -37,11 +37,15 @@ class OgSelection extends DefaultSelection {
    * @return DefaultSelection
    */
   public function getSelectionHandler() {
-    $field = $this->getConfiguration('field');
-
-    if ($field) {
-      return \Drupal::service('plugin.manager.entity_reference_selection')->getSelectionHandler($field);
```

Comment:
@amitaibu I have a clean install from earlier (don't have time to rebuild now) and it was ok. This PR shouldn't affect that at least.